src/test/java/cart/domain/CartItemTest.java

```diff
@@ -0,0 +1,45 @@
+package cart.domain;
+
+import cart.exception.CartItemException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
```

Comment:
R : 한글로 메서드 작성할 때 `_`를 지워서 가독성을 높일 수 있더라고 !

```suggestion
@DisplayNameGeneration(ReplaceUnderscores.class)
```

Reply:
고맙습니당 반영해뒀어!